frontend/src/views/Raid.vue

```diff
@@ -385,89 +423,89 @@ export default {
       });
     },
 
-    promptRewardClaim() {
+    promptRewardClaim(): void {
       // should offer a popup here to pick which index to claim
       // if only one index, then claim instantly
       if(this.rewardIndexes !== null && this.rewardIndexes.length > 0) {
         if(this.rewardIndexes.length === 1) {
           this.claimRewardIndex(this.rewardIndexes[0]);
         }
         else {
-          this.$bvModal.show('rewardsRaidPicker');
+          (this as any).$bvModal.show('rewardsRaidPicker');
         }
       }
     },
 
-    async claimRewardIndex(rewardIndex) {
+    async claimRewardIndex(rewardIndex: string): Promise<void> {
       this.bonuxXpCharacterNames = [];
       this.bonuxXpAmounts = [];
       const result = await this.claimRaidRewards({
         rewardIndex
       });
 
-      const nfts = [];
+      const nfts: Nft[] = [];
```

Comment:
This type should be NftIdType[]. Actual Nfts are fetched in NftList component based on id and type. While this will work as is, I think it might introduce some confusion.

Here's a prop in NftList.vue that the collection created here is passed to:
```
nftIdTypes: {
  type: Array as PropType<NftIdType[]>,
  default() {
    return [];
  },
},
```